modules/ipc/methods/eth_sendTransaction.js

```diff
@@ -36,14 +36,16 @@ module.exports = class extends BaseProcessor {
             try {
                 _.each(payload.params[0], (val, key) => {
                     // if doesn't have hex then leave
-                    if (_.isString(val)) {
+                    if (_.isString(val) || _.isNumber(val)) {
```

Comment:
Won't this throw when you try to do `toLowerCase()` on the number a few lines down?